### ISSUE TITLE
Need to specify particular params for mock call

### ISSUE THREAD

**Issue by cakeinpanic** [NagRock/ts-mockito#85]
###When I have updated from `2.3` to `2.9` I found out that these constructions don't work anymore:

```
const testServiceMock = mock<TestService>(TestService);
when(testServiceMock.hello(anything())).thenReturn('a')

const testService = new TestService();
testService.hello(); // mock won't work cause I specified that it should work only when 1 argument is passed
testService.bye(); // mock won't work cause i haven't specified it at all
```

Ability to use `anything()` for not provided parameters was great(and totally in js way), why did you took it away?(

**Comment by ryuuji3** [comment]
It didn't go anywhere, I don't think:
https://github.com/NagRock/ts-mockito/blob/master/src/matcher/type/AnythingMatcher.ts

May have been since updated.

**Comment by cakeinpanic** [comment]
you didn't get me right)
matcher is still the same, the way number of method arguments is processed has changed, previously(in 2.3) if you write 
`when(testServiceMock.hello(anything())).thenReturn('a')`
you got matched for method calls:
```
testService.hello();
testService.hello(123);
```
and since 2.9 only for
```
testService.hello(123);
```

**Comment by ryuuji3** [comment]
Ah my bad. I'm unable to test this at the moment but I think
```ts
when(testServiceMock.hello).thenReturn('a')
```
may be a quick fix for now. Let me know if that works.

**Comment by cakeinpanic** [comment]
yes, i have already moved _all_ our tests on this flow as soon as discovered it

but I think that possibility to use `anything()` as no parameter at all is very helpful and should be returned back)
